SeventhHeavenUI/Windows/ConfigureGLWindow.xaml.cs

```diff
@@ -56,6 +56,21 @@ public bool Init(string cfgSpec, string cfgFile)
                 return false;
             }
 
+            // build new DDOption for resolutions
+            List<Iros._7th.Workshop.ConfigSettings.DDOption> DigitalResolutions = new List<Iros._7th.Workshop.ConfigSettings.DDOption>();
+            DigitalResolutions.Add(new Iros._7th.Workshop.ConfigSettings.DDOption() { Settings = "window_size_x = 0,window_size_y = 0", Text = "Auto" });
+
+            List<SupportedResolution> supportedResolutions = PrimaryScreen.GetSupportedResolutions();
+            foreach(SupportedResolution sr in supportedResolutions)
+            {
+                DigitalResolutions.Add(new Iros._7th.Workshop.ConfigSettings.DDOption() { Settings = $"window_size_x = {sr.H},window_size_y = {sr.V}", Text = $"{sr.H}x{sr.V}"});
+            }
+
+            // find the resolutions option
+            Iros._7th.Workshop.ConfigSettings.DropDown ResolutionDD = (Iros._7th.Workshop.ConfigSettings.DropDown)_spec.Settings.Find(itm => { return itm.Name == "Resolution"; });
```

Comment:
Can you please rename `itm` into `item`? I know it's not critical but one byte more won't kill anyone and still improve the readability :D

Reply:
Done, I had to merge changes from other fixes into this branch to get it to work so that has also been done. so now it should squash and merge correctly